doc/app_config.md

```diff
@@ -50,3 +56,100 @@ section-1: value-1  # hasn't been changed
 section-2:
   key-21: value-21-new  # body was replaced
 ```
+
+## Config uploading
+
+In contrast to using `cartridge_app_config`, `cartridge_app_config_path`
```

Comment:
Seems strange. Maybe just the "`cartridge_upload_app_config` step (link to step doc) allows loading the entire config using HTTP endpoint or Lua API."?